### PR TITLE
Send copy of booking email to prison

### PR DESCRIPTION
**Pull request by threedaymonk**
It is useful for the prison to have a copy of the information sent in order to resolve disputes (especially around what was or was not communicated to the visitor).

### CODE REVIEW

File: config/locales/en/mailers.yml [hunk 1]
```diff
@@ -17,6 +17,11 @@ en:
         If you cannot click on the link above, copy the entire link and paste
         it into the address bar of your browser.
       visit_id: 'Visit ID:'
+    booked:
+      subject: >
+        COPY of booking confirmation for Oscar Wilde
```
Comment (tatyree) [inline config/locales/en/mailers.yml]: You are all Oscar Wilde now...

File: spec/features/process_a_request_spec.rb [hunk 2]
```diff
@@ -3,13 +3,22 @@
...
+  let(:visitor_email_address) { 'visitor@test.example.com' }
+  let(:prison_email_address) { 'prison@test.example.com' }
+  let(:prison) {
+    create(
+      :prison,
+      name: 'Reading Gaol',
+      email_address: prison_email_address
+    )
+  }
   let(:vst) {
```
Comment (tatyree) [inline spec/features/process_a_request_spec.rb]: Some obscure military aircraft from the 70's perhaps? :smile:

Comment (threedaymonk) [inline spec/features/process_a_request_spec.rb]: Or digital audio processing plugins: https://en.wikipedia.org/wiki/Virtual_Studio_Technology

File: config/locales/en/mailers.yml (hunk 1 above)
Comment (threedaymonk) [inline config/locales/en/mailers.yml]: Oops!

File: spec/features/process_a_request_spec.rb (hunk 2 above)
Comment (threedaymonk) [inline spec/features/process_a_request_spec.rb]: More seriously, `visit` is already taken by Capybara for visiting a page. An unfortunate clash.

**Pull request merged**